src/x/panicmon/executor_test.go

```diff
@@ -287,6 +287,7 @@ func TestExecutorPassSignals(t *testing.T) {
 	procInfo.Signal(syscall.SIGUSR1)
 
 	child.Wait()
+	ex.closeAndWait()
```

Comment:
Oh this was flaking?

Reply:
Yeah its been flaking for a while, but I fixed it in another P.R. Let me rebase and this change will go away